.travis.yml

```diff
@@ -35,8 +31,7 @@ before_script:
   - composer update
 
 script: phpunit --coverage-text && phpunit Tests/Controller/SecurityTest.php
```

Comment:
you should rather have 2 different scripts rather than running 2 commands with ``&&`` IMO